.github/workflows/test-publish-dry-run.yaml

```diff
@@ -72,5 +72,14 @@ jobs:
       - name: Set Gradle Version
         run: ./gradlew wrapper --gradle-version $VERSION_GRADLE --distribution-type=all
 
-      - name: Run the persistSentryProguardUuids task
-        run: ./gradlew assembleRelease persistSentryProguardUuidsForReleaseRelease
+      - name: Run the uploadSentryProguardMappings task
+        run: ./gradlew assembleRelease uploadSentryProguardMappingsReleaseRelease
+
+      - name: Check sentry-debug-meta.properties inside APKs
+        run: |
+          unzip ./examples/android-gradle/build/outputs/apk/release/android-gradle-release-unsigned.apk -d output
+          cat output/assets/sentry-debug-meta.properties
```

Comment:
does that mean if `cat` fails (probably file not found), the build fails? :) awesome

Reply:
Yes :)